### PR TITLE
ToU API updates for versioning

### PR DESCRIPTION
**Pull request by rajadineshmurugesan-microsoft**

### CODE REVIEW

**Comment by Lauragra** [comment]
@rajadineshmurugesan-microsoft , please follow the get started steps to get set up as a contributor to the repo: https://msazure.visualstudio.com/One/_wiki/wikis/Microsoft%20Graph%20Partners/55085/Get-started-with-GitHub. Please also review the workflow topic and set labels on the PR as appropriate: https://msazure.visualstudio.com/One/_wiki/wikis/Microsoft%20Graph%20Partners/55055/Content-workflow.

File: api-reference/beta/resources/agreement.md [hunk 1]
```diff
@@ -45,7 +45,9 @@ Represents a tenant's customizable terms of use agreement that is created and ma
 |:-------------|:------------|:------------|
 |acceptances|[agreementAcceptance](agreementacceptance.md) collection|Read-only. Information about acceptances of this agreement.|
 |files|[agreementFileLocalization](agreementfilelocalization.md) collection| PDFs linked to this agreement. **Note:** This property is in the process of being deprecated. Usethe  **file** property instead.|
```
Comment (raprakasMSFT) [inline api-reference/beta/resources/agreement.md:47]: please add a space between 'Use' and 'the'

**Review by raprakasMSFT — APPROVED** [review]
+1 pending comments

File: api-reference/beta/resources/agreement.md [hunk 2]
```diff
@@ -44,8 +44,10 @@ Represents a tenant's customizable terms of use agreement that is created and ma
 | Relationship | Type        | Description |
 |:-------------|:------------|:------------|
 |acceptances|[agreementAcceptance](agreementacceptance.md) collection|Read-only. Information about acceptances of this agreement.|
-|files|[agreementFileLocalization](agreementfilelocalization.md) collection| PDFs linked to this agreement. **Note:** This property is in the process of being deprecated. Usethe  **file** property instead.|
-|file|[agreementFile](agreementfile.md) | PDFs linked to this agreement.|
+|files|[agreementFileLocalization](agreementfilelocalization.md) collection| PDFs linked to this agreement. **Note:** This property is in the process of being deprecated. Use the  **file** property instead.|
+|file|[agreementFile](agreementfile.md) | Default PDF linked to this agreement.|
+|file/localizations|[agreementFileLocalization](agreementfilelocalization.md) collection|The localized versions of the agreement files attached to the agreement.|
+|file/localizations/{localizationId}/versions|[agreementFileVersion](agreementfileversion.md) collection|The version history for the localized agreement file.|
```
Comment (Lauragra) [inline api-reference/beta/resources/agreement.md:50]: I'm not sure about how we're referencing these relationship properties - this is not the standard structure. Where is the CSDL that shows this?

File: concepts/permissions-reference.md [hunk 3]
```diff
@@ -2304,4 +2304,4 @@ This section shows some common scenarios that target [user](/graph/api/resources
 | App wants to read all content in all Microsoft 365 groups, including files, conversations.  It also needs to show group memberships, be able to update group memberships, (if owner).  |  _Group.Read.All_ | Read items in all site collections, Read all groups|
 | App wants to read and write all content in all Microsoft 365 groups, including files, conversations.  It also needs to show group memberships, be able to update group memberships, (if owner).  | 	_Group.ReadWrite.All_, _Sites.ReadWrite.All_ |  Read and write all groups, Edit or delete items in all site collections |
 | App wants to discover (find) a Microsoft 365 group. It allows the user to search for a particular group and choose one from the enumerated list to allow the user to join the group.	 | _Group.ReadWrite.All_ | Read and write all groups|
-| App wants to create a group through AAD Graph | 	_Group.ReadWrite.All_ | Read and write all groups|
\ No newline at end of file
+| App wants to create a group through AAD Graph | 	_Group.ReadWrite.All_ | Read and write all groups|
```
Comment (Lauragra) [inline concepts/permissions-reference.md:2307]: AAD -> Azure AD

File: api-reference/beta/resources/agreementfileversion.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+# agreementFileVersion resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a customized version of terms of use agreement file that a tenant manages with Azure Active Directory (Azure AD). It contains metadata about the agreement file (for example, the name, the language, and whether it is the default file).
+
+<!--
```
Comment (Lauragra) [inline api-reference/beta/resources/agreementfileversion.md:18]: Why are sections getting commented out in the file?

**Review by Lauragra — CHANGES_REQUESTED** [review]
This PR will require an update to the changlog MD and JSON files.

File: api-reference/beta/resources/agreement.md (hunk 2 above)
Comment (rajadineshmurugesan-microsoft) [reply api-reference/beta/resources/agreement.md:50]: We are following the tenant branding pattern for 'localizations'. That's why it's not been shown as a relationship in the 'agreementFile' file. So, we wanted to explicitly mention this in here.

File: concepts/permissions-reference.md (hunk 3 above)
Comment (rajadineshmurugesan-microsoft) [reply concepts/permissions-reference.md:2307]: Oh. This wasn't our change. Nothing is changed here. I reverted some changes in this file, but still this is showing up.

File: api-reference/beta/resources/agreementfileversion.md (hunk 4 above)
Comment (rajadineshmurugesan-microsoft) [reply api-reference/beta/resources/agreementfileversion.md:18]: Copy/Paste issue. Can remove it if needed.

**Comment by rajadineshmurugesan-microsoft** [comment]
Added the changelog

**Review by Lauragra — APPROVED** [review]

**Pull request merged**